src/routes/plans/[id]/+page.svelte

```diff
@@ -143,6 +145,34 @@
     planHasBeenLocked = false;
   }
 
+  $: schedulingAnalysisStatus = $schedulingStatus;
+
+  $: if ($schedulingSpecGoals) {
+    schedulingGoalCount = 0;
```

Comment:
Would it make sense to keep these stats in a derived store?

Reply:
Yeah, I'll move it to one.